commit 190b8bbe7d94fe72636380d20585a0c7a6aec54c (refs/changes/58/4858/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-09-25T15:55:04+02:00 (1 year ago)
    
    TRITON-785 Add requirements.brand and requirements.bootrom to AdminUI's images view
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/www/js/tpl/image.hbs b/www/js/tpl/image.hbs
index e5bb027f..449fa510 100644
--- a/www/js/tpl/image.hbs
+++ b/www/js/tpl/image.hbs
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright (c) 2018, Joyent, Inc.
 -->
 
 <div class="page-header">
@@ -240,6 +240,20 @@
         </span>
       </li>
       {{/if}}
+
+      {{#if requirements.bootrom}}
+      <li>
+        <strong>BootROM</strong>
+        <span class="value">{{requirements.bootrom}}</span>
+      </li>
+      {{/if}}
+
+      {{#if requirements.brand}}
+      <li>
+        <strong>Brand</strong>
+        <span class="value">{{requirements.brand}}</span>
+      </li>
+      {{/if}}
     </ul>
     </section>
 
